### PR TITLE
Update tutorial translations

### PR DESCRIPTION
**Pull request by mmilata**
Followup of #3982.

### CODE REVIEW

**Comment by mmilata** [comment]
UI reports:
* https://data.trezor.io/dev/firmware/ui_report/9848940578/T3T1-cs-core_device_test/index.html
* https://data.trezor.io/dev/firmware/ui_report/9848940578/T3T1-fr-core_device_test/index.html
* https://data.trezor.io/dev/firmware/ui_report/9848940578/T3T1-de-core_device_test/index.html
* https://data.trezor.io/dev/firmware/ui_report/9848940578/T3T1-es-core_device_test/index.html

File: core/translations/cs.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
   },
   "header": {
     "language": "cs-CZ",
-    "version": "2.8.0"
+    "version": "2.8.1"
```
Comment (Hannsek) [inline core/translations/cs.json:36]: Shouldn't it be 2.8.0 still?

Comment (matejcik) [reply core/translations/cs.json:36]: nah, we need the translations matched to the current in-tree version.

we can generate 2.8.0 blobs regardless of what is written here

**Review by matejcik — APPROVED** [review]
LGTM. I pushed de332bdd3887919e67f05ab7bb17ed20cc4bf6c1 to simplify this in the future, ptal @mmilata

**Pull request merged**